packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react.test.ts.snap

```diff
@@ -44,7 +44,172 @@ export default function CreateCustomerButton(
 }
 `;
 
-exports[`amplify render tests actions navigation action 1`] = `
+exports[`amplify render tests actions auth signs out 1`] = `
+Object {
+  "componentText": "/* eslint-disable */
+import React from \\"react\\";
+import {
+  EscapeHatchProps,
+  getOverrideProps,
+  useAuthSignOutAction,
+} from \\"@aws-amplify/ui-react/internal\\";
+import { Button, ButtonProps } from \\"@aws-amplify/ui-react\\";
+
+export type SignOutButtonProps = React.PropsWithChildren<
+  Partial<ButtonProps> & {
+    overrides?: EscapeHatchProps | undefined | null;
+  }
+>;
+export default function SignOutButton(
+  props: SignOutButtonProps
+): React.ReactElement {
+  const { overrides: overridesProp, ...rest } = props;
+  const overrides = { ...overridesProp };
```

Comment:
Similar change from other PR required here as well to avoid this unnecessary renaming?